### DISCUSSION TITLE
Is there a possibility to change the graphics of the Project?

### DISCUSSION THREAD

**Discussion by Suiky-DD** [Blackymas/NSPanel_HA_Blueprint#1287] [Q&A]
I think when i understood right, its the tft that defines the style of the project, or am i wrong. I tried to open the document with different programs, but it didnt work. So i would like to ask, if i can configure or style the design in any way. Im a Designer / Web Programmer, so i think if you would lead my way to a solution, i could do something. Maybe others are also interessted in different design Layouts etc.

**Comment by edwardtfn** [comment]
That would be awesome.
You are right, all the design is on the TFT file, which is installed directly in the display (the touch screen itself) and we use ESPHome to send commands and values for those components.
The TFT file is generated by [Nextion Editor](https://nextion.tech/nextion-editor/), where you can also edit those pages. The source file (editable) is the `.hmi` you will find in the repository.
We try to keep all the functionalities on both EU and US version, to the level of having same ids for componentes etc. Só if you wanna share your changes, it would be nice if you play with both models. The US landscape is just the EU version with a couple of modifications that you don't have to worry about, but you can find listed [here](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/nspanel_us_land_code/instructions.md).

Please let me know if you get stuck.

**Reply by edwardtfn** [reply]
I can say that I have my own customization which consists on making the outside temperature bigger, but it's a bit of work to keep it updated, so I don't have that as an official TFT published in this repo, but have shared with other people using my own fork.
